composer.json

```diff
@@ -32,8 +32,8 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Google\\": "src/Google/Ads/GoogleAds/Util/Testing/Google",
-      "GPBMetadata\\": "src/Google/Ads/GoogleAds/Util/Testing/metadata"
+      "Google\\Ads\\GoogleAds\\": "tests/Google/Ads/GoogleAds",
+      "GPBMetadata\\Google\\Ads\\GoogleAds\\": "tests/metadata/Google/Ads/GoogleAds"
```

Comment:
Just in case: This doesn't have bad effect with normal source and example files, right?

Reply:
Not that I could find, at least `GetCampaigns.php` runs fine. Any particular reason you think it could have an impact?

Reply:
They're the same as those rules in "psr-4" of "autoload", so I just wonder if they play along together well.
Previously, they're different rules.

Reply:
I think the best is to regroup all testing classes that are not actual tests under the package `Google\Ads\GoogleAds\Tests` as suggested by Composer `autoload-dev` documentation [here](https://getcomposer.org/doc/04-schema.md#autoload-dev). What do you think?

Reply:
I don't think that doc recommends grouping all non-test into `Tests`, but grouping any helping files into one place would be fine, of course.
However, as I mentioned in the comment thread below, you probably don't need to do that, as we can specify a suffix in the rule.

So, let's go with this version you have.

Reply:
I will wait for the code coverage report to be updated following my last commit to be sure. If there is a drop I suggest that we regroup those helper classes in this single package and add a rule to whitelist exclude it from the phpunit coverage.

Reply:
The code coverage changed but it is unrelated to the change so I keep it like that for now. We can further improve this later on if needed.